tests/utils/autoTester.js

```diff
@@ -212,7 +243,7 @@ module.exports.setupTest = function (primaryCamera) {
     spectatorCameraConfig.resetSizeSpectatorCamera(1920, 1080);
     spectatorCameraConfig.vFoV = 45;
     Render.getConfig("SecondaryCameraJob.ToneMapping").curve = 0;
-	  Render.getConfig("SecondaryCameraJob.DrawHighlight").enabled = false;
+	Render.getConfig("SecondaryCameraJob.DrawHighlight").enabled = false;
```

Comment:
Bad indentation at line 246.  Two spaces were removed but only one should have been.